### DISCUSSION TITLE
Q/A: Do you have access to github from your z/OS machines

### DISCUSSION THREAD

**Discussion by IgorTodorovskiIBM** [zopencommunity/meta#24] [Polls]
Some vendors block github.com access.  We're trying to understand how pervasive this is.

**Comment by lbdyck** [comment]
I would suggest you do this survey on the ibm-main listserv where you'll get more traction.

**Reply by lbdyck** [reply]
Suggest sending the question out with a link to survey monkey so you can easily collect the results.

**Reply by IgorTodorovskiIBM** [reply]
That's a good suggestion, thank you. I don't think we've engaged with the ibm-main listserv community yet, so thanks!

**Comment by lbdyck** [comment]
See https://www.surveymonkey.com/r/MT3G3SK for the survey.  Posted to ibm-main, twitter, and the Mainframe Facebook group.  Also posted to the IBM Champions slack space and the Open Mainframe Project slack space.

**Comment by MikeFultonDev** [comment]
@lbdyck now that our community has grown, should we do the poll again? I'd love to see if we can provide really good 'air gapped' support

**Reply by IgorTodorovskiIBM** [reply]
@DevonianTeuchter is working on an air-gapped solution in zopen. It may be a good topic for a blog and/or a demo for the next guild meeting.

**Reply by DevonianTeuchter** [reply]
@MikeFultonDev To expand on @IgorTodorovskiIBM 's comment.
I've got a custom zopen install running with a --paxinstall option,  installing the pax.Z file(s) directly. It relies on the internal metadata and the name of the pax for some information and checks in ./ or uses a custom directory via a --pax-repo-dir parameter on the install command or through a system setting from zopen init (which sets /etc/zopen/paxrepodir).  [The pax gets copied to the install cache and from then on it is a "normal" install - it was just the initial location/metadata retrieval that needs implemented]
It might be nice if there was a single metadata/spec file in the pax that could be referenced. If that was in JSON, we could analyse the pax files and generate a repo-metadata file - using JSON would look similar to the data currently pulled from the repo cache already.
What about having a zopen repo/mirror/repobuild/reposync/... command that would pull new port versions from GH to a custom location and generate suitable repo metadata (or pull the port info from the pax as above to generate the repo)..?
